### DISCUSSION TITLE
Go To Bank is replaced by Switch ON when sending to device

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#70] [Editor App Feature Requests] [answered]
When programming a switch to do a bankjump:
Type: Smart Message
Function: Go To Bank#

When trying to send the file to the unit, the function converts to 
Function: Switch On

...making bank jumps not work.

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by toreriksolem** [comment]
I also tried to program different smart messages: Bank up and bank down. They both reverted to Switch On on the Bridge6

**Comment by konnsim** [comment]
Issue with device firmware, will be solved in Beta9 release.

**Comment by samspencer5991** [comment]
This is related to #75 and should be fixed in the latest v1.1.1-beta1 release. I'll wait for you @Konnsalad to confirm

**Comment by konnsim** [comment, marked as answer]
Fixed on firmware v1.1.1-beta.4.
